### ISSUE TITLE
Debugging on Raspberry Pi 2 throws exception

### ISSUE THREAD

**Issue by Kinani** [sensaura-public/iotweb#7]
Debug output:
Exception thrown: 'System.Runtime.InteropServices.COMException' in mscorlib.ni.dll
WinRT information: An attempt was made to access a socket in a way forbidden by its access permissions.

I have the required Capabilities checked on appxmanifest:
Internet(Client & Server)
Internet(Client)
Private Networks(Client & Server)

Any idea how to fix this?

**Comment by deandob** [comment]
Yes - likely the Windows firewall, I had the same problem with my RPi3 when using websockets, and I changed the port to 80 and could get the webserver working but unfortunately the websockets fails to work on RPi when the same code works fine on a PC. I don't get any error messages just the thread exits.

Any ideas?
